dig/node.go

```diff
@@ -60,13 +60,11 @@ func (n node) id() string {
 
 type objNode struct {
```

Comment:
shouldn't refactor now, because it will probably change very soon, but having a struct that only embeds one other struct and nothing else looks odd :)